doc/governance/index.rst

```diff
@@ -0,0 +1,72 @@
+Ceph Governance
+---------------
+
+Key principles
+==============
+
+ * Decision-making is consensus-driven by those who show up.
+ * Leadership roles are defined primarily by responsibility, not prestige or seniority.
+   * It is normal and healthy for these roles to be passed on to others
+ * Everyone's role is ultimately to serve the users and participation
+   is voluntary.
+
+Bodies
+------
+
+Ceph Executive Council
+======================
+
+Responsibilities
+^^^^^^^^^^^^^^^^
+
+ * Arbiter in cases where decisions cannot be reached by consensus
+ * Distribute key responsibilities amongst themselves or others
+ * Point of contact for the project
+ * Representatives for Ceph foundation board meetings
+ * Ensure things get done
+
+Membership
+^^^^^^^^^^
+
+ * 3 people
```

Comment:
> LGTM! Just left a few comments and suggestions over there.
> 
> Some other things (mostly from having a look at [Python's](https://peps.python.org/pep-0013/) and [Mozilla's](https://www.mozilla.org/en-US/about/governance/) ones):
> 
> * Comparing to Mozilla's, we would have defined here the "Roles & Responsibilities" and the "Organizations" aspects, but 2 other topics would remain blurry/undefined ("Policies", and "Dicussion Forums/Communication").
> * **Roles & Responsibilities**:
>   
>   * This is perhaps premature, but have we planned how to deal with 'power vacuums' if a/some members have to abruptly step down (life happens: sick/parental leaves, etc). (e.g.: [Python's example](https://peps.python.org/pep-0013/#vacancies)).

Not exactly, python's example of appointing someone, or immediately doing an election, both seem viable.

>   * Should we clarify the nomination processes? Just as an example [this is how it's explained in the Python Community Governance](https://peps.python.org/pep-8012/#nominating-a-new-core-developer).

Agreed, we should add some more detail about expectations here

> * **Policies**:
>   
>   * Who's in charge of enforcing our [Code of Conduct](https://docs.github.com/en/communities/setting-up-your-project-for-healthy-contributions/adding-a-code-of-conduct-to-your-project) (I assume we adhere at least to [Linux Foundation's one](https://events.linuxfoundation.org/about/code-of-conduct/))? For example: as a project we are committed to remove racially insensitive terms from the code, but I cannot infer that commitment from anything stated here.

Good point, we have the code of conduct on the website (https://ceph.io/en/code-of-conduct/) but not mentioned in the docs. We could also mention making contributing accessible and inclusive like Python does - it looks like a lot of thought went into writing those documents.

>   * What about conflicts of interest, mainly due to _same_ vendor prevalence? We talked about ensuring that not all Council members worked for the same employeer, but there's no mention to that. ([Python's](https://peps.python.org/pep-0013/#conflicts-of-interest)).

Requiring more than 1 organization be represented on the council would be a good start.

> * **Communication**:
>   
>   * While this is mostly an operational concern, I'd ensure that we define here clear points of contact for each governing body, so everyone reading this knows exactly how to address to each one (and not having to maintain those listings in many places, since sooner or later they'll become outdated). Examples:
>     
>     * **Council**: [council@ceph.io](mailto:council@ceph.io) or Github's @ceph/council
>     * **Ceph Steering Committee (CSC?)**: [csc@ceph.io](mailto:csc@ceph.io) or Github's @ceph/csc

Sure, this seems like a reasonable place to include these. Thinking just email, since github shouldn't be a requirement for being part of the steering commiittee.